src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,54 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+
+    if (!health) {
+      this.health = 100;
+    } else {
+      this.health = health;
+    }
```

Comment:
```suggestion
    this.health = 100;
    }
```
У всех по дефолту 100, параметр здоровья не передается

Reply:
Исправил. Просто почему-то подумал, что пользователь может создавать зверя со своим уникальным здоровьем)